### PR TITLE
PR: Automatic "on-the-fly" code completion (i.e. without using Tab)

### PR DESCRIPTION
**Pull request by malliwi88**

### CODE REVIEW

File: spyder/widgets/sourcecode/base.py [hunk 1]
```diff
@@ -49,7 +49,7 @@ def insert_text_to(cursor, text, fmt):
 
 
 class CompletionWidget(QListWidget):
-    """Completion list widget"""
+    """ Completion list widget. """
```
Comment (goanpeca) [inline spyder/widgets/sourcecode/base.py]: No spaces before or after.

`"""Completion list widget."""`

File: spyder/widgets/sourcecode/codeeditor.py [hunk 2]
```diff
@@ -2758,6 +2758,10 @@ def keyPressEvent(self, event):
             TextEditBaseWidget.keyPressEvent(self, event)
             if self.is_completion_widget_visible() and text:
                 self.completion_text += text
+            else:
+                # launch inline completion
```
Comment (goanpeca) [inline spyder/widgets/sourcecode/codeeditor.py]: Launch inline completion

**Comment by andfoy** [comment]
This PR should be closed, as #4751 implements it

**Comment by goanpeca** [comment]
@ccordoba12 should we close this?

**Comment by ccordoba12** [comment]
Yes, it doesn't apply anymore.

**Pull request closed without merging**